projects/plugins/jetpack/extensions/blocks/tock/edit.js

```diff
@@ -44,42 +48,60 @@ const UrlPopover = ( { tockUrl, setEditedUrl, popoverAnchor, setUrl, cancel } )
 		}
 	};
 
+	const closeIfNotToggle = () => {
```

Comment:
This is all a bit messy, but we essentially have to implement our own dropdown so that we can call our own reset code on close, but only when the popover is closed by losing focus.